golang/cosmos/app/upgrade.go

```diff
@@ -238,6 +238,19 @@ func makeUnreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string) upgradetyp
 			}
 		}
 
+		// Upgrade vat-orchestration to avoid noisy rejections.
```

Comment:
isn't `local-orchestration-account.js` library code?

I don't see it in the `vat-orchestration` bundle:

```console
~/projects/agoric-sdk/packages/boot$ jq -r '.contents | .[] | .relativePath' bundles/bundle-vat-orchestration-js-meta.json|grep local-orch
```

I see various other stuff...

```console
~/projects/agoric-sdk/packages/boot$ jq -r '.contents | .[] | .relativePath' bundles/bundle-vat-orchestration-js-meta.json|head -4

../exos/cosmos-interchain-service.js
../../../../node_modules/@endo/far/src/index.js
../../../vow/vat.js
```

Reply:
`local-orchestration-account.js` is, however, used to make the contract bundle:

```console
~/projects/agoric-sdk/packages/portfolio-deploy$ jq -r '.inputs | keys | .[]' dist/portfolio.contract.meta.json |grep local-orch
../orchestration/src/exos/local-orchestration-account.js
```

Reply:
I found that the change I introduced is present in a build of `agoric-sdk/packages/portfolio-deploy`:

```
cd agoric-sdk/packages/portfolio-deploy
yarn build
[...]
grep extractFirstQueryResult dist/portfolio.contract.bundle.js
```

No need for an `x/gov` chain software-upgrade!